internal/pkg/levee/testdata/src/example.com/tests/propagation/tests.go

```diff
@@ -20,21 +20,77 @@ import (
 	"example.com/core"
 )
 
-func Identity(arg interface{}) interface{} {
-	return arg
+func main() {
```

Comment:
Do you think these new tests are related to propagation?

I think we are testing something else here, but I can't quite sure how to name this. WDYT?

Reply:
Good question. I think propagation may be too broad. Indeed, *taint propagation* is the whole point of our analysis. Maybe propagat***ors*** would be more appropriate?